docs/source/en/api/pipelines/chroma.md

```diff
@@ -27,9 +27,36 @@ Chroma can use all the same optimizations as Flux.
 
 </Tip>
 
-## Inference (Single File)
+## Inference
 
-The `ChromaTransformer2DModel` supports loading checkpoints in the original format. This is also useful when trying to load finetunes or quantized versions of the models that have been published by the community.
+The Diffusers version of Chroma is based on the `unlocked-v37`  version of the original model, which is available in the [Chroma repository](https://huggingface.co/lodestones/Chroma).
+
+```python
```

Comment:
Maybe we could discard it and make it a part of the pipeline example docs? 

And then we could just include the `from_single_file()` example.

Reply:
Sounds good to me.